base/memory/discardable_shared_memory.cc

```diff
@@ -379,21 +381,21 @@ void DiscardableSharedMemory::Unlock(size_t offset, size_t length) {
     return;
 
   Time current_time = Now();
-  DCHECK(!current_time.is_null());
+  CHECK(!current_time.is_null());
```

Comment:
This CHECK and the one below need to be converted back to DCHECK.  This bloats our diff vs upstream chromium, adds overhead, and may contribute to instability.